scripts/changelog.js

```diff
@@ -51,14 +51,15 @@ const formatChanges = (changes) =>
         .replace(/[a-z]+\((\w+)\)/, '$1')
         // Linkify commit refs.
         .replace(
-          /^(\w{8})/,
+          /^(\w+)\s/,
```

Comment:
I don’t think we want to replace the whitespace here.
```suggestion
          /^(\w+)/,
```

Reply:
You're right, in fact I was re-introducing that space a couple of lines later. Fixed it, thanks!